File: JetpackProject/app/src/main/java/com/cyq/jetpack/MainActivity.java
package com.cyq.jetpack;

import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;
import androidx.databinding.ViewDataBinding;

import android.os.Bundle;

import com.cyq.jetpack.databinding.ActivityMainBinding;
import com.cyq.jetpack.model.User;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DataBindingUtil.setContentView(this, R.layout.activity_main);
        User user = new User("张三", 40);
        binding.setUser(user);


    }
}

File: JetpackDemo/Demo/app/src/main/java/com/chenyangqi/demo/MainActivity.kt
package com.chenyangqi.demo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.coroutines.launch
import kotlinx.coroutines.supervisorScope

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    suspend fun test() {
        supervisorScope {
            launch { }
            launch { }
        }
    }
}